compiler/tflchef/tflite/src/Op/HardSwish.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
2023?

Reply:
Oh, copy-paste typo.
Thanks for finding it.